frameworks/cassandra/universe/config.json

```diff
@@ -730,6 +735,116 @@
+        },
+        "windows_timer_interval": {
+          "type": "integer",
+          "description": "The default Windows kernel timer and scheduling resolution is 15.6ms for power conservation. Lowering this value on Windows can provide much tighter latency and better throughput, however some virtualized environments may see a negative performance impact from changing this setting below their system default.",
+          "default": 1
+        },
+        "counter_cache_save_period": {
+          "type": "integer",
+          "description": "the amount of time after which Cassandra saves the counter cache (keys only).",
+          "default": 7200
+        },
+        "trickle_fsync_interval_in_kb": {
+          "type": "integer",
+          "description": "The size of the fsync in kilobytes.",
+          "default": 10240
+        },
+        "trickle_fsync": {
+          "type": "boolean",
+          "description": "When set to true, causes fsync to force the operating system to flush the dirty buffers at the set interval ",
+          "default": false
+        },
+        "incremental_backups": {
+          "type": "boolean",
+          "description": "Backs up data updated since the last snapshot was taken. When enabled, Cassandra creates a hard link to each SSTable flushed or streamed locally in a backups subdirectory of the keyspace data.",
+          "default": false
+        },
+        "snapshot_before_compaction": {
+          "type": "boolean",
+          "description": "Enables or disables taking a snapshot before each compaction. A snapshot is useful to back up data when there is a data format change.",
+          "default": false
+        },
+        "cross_node_timeout": {
+          "type": "boolean",
+          "description": "operation timeout information exchange between nodes (to accurately measure request timeouts).",
+          "default": false
+        },
+        "role_manager": {
+          "type": "string",
+          "description": "Part of the Authentication & Authorization backend that implements IRoleManager to maintain grants and memberships between roles.",
+          "default": "CassandraRoleManager"
+        },
+        "commit_failure_policy": {
+          "type": "string",
+          "description": "Policy for commit disk failures.",
+          "default": "stop"
+        },
+        "key_cache_size_in_mb": {
+          "type": "string",
+          "description": "A global cache setting for the maximum size of the key cache in memory (for all tables). ",
+          "default": ""
+        },
+        "counter_cache_size_in_mb": {
+          "type": "string",
+          "description": "When no value is set, Cassandra uses the smaller of minimum of 2.5% of Heap or 50MB.",
+          "default": ""
+        },
+        "commitlog_sync": {
+          "type": "string",
+          "description": "The method that Cassandra uses to acknowledge writes in milliseconds",
+          "default": "periodic"
```

Comment:
is there a finite set of possible configs? I would suggest using enums in this case

Reply:
done